packages/apollo-engine-reporting/src/__tests__/extension.test.ts

```diff
@@ -65,11 +65,17 @@ test('trace construction', async () => {
     addTrace,
```

Comment:
How about end-to-end tests! Search for eg `sets the trace key to operationName when it is defined`